### ISSUE TITLE
BorderlessMC automatic borderless not making windows borderless with MultiMC

### ISSUE THREAD

**Issue by cornerkey** [Mr-Technician/BorderlessMinecraft#34]
The Latest release (v1.3.2 ) fixed the issue of BorderlessMC crashing on instance start from MultiMC if automatic borderless was enabled
I've encountered another proble i.e :
 I have automaticborderless enabled but borderlessMC just doesnt make minecraft borderless when it starts it just stays windowed and i have to manually make it borderless. 
The reason i require automatic borderless is bc i am writing up a python script that open a complete catalogue of programs and borderlessmc are one of them

**Comment by Mr-Technician** [comment]
Oh no, I was positive I fixed this. 😭

**Comment by Mr-Technician** [comment]
In my limited testing it seems auto borderless is hit or miss with multiMC. This is probably related to the initial issue of how multiMC starts the java process and how that affects the process detection. I will need to investigate further.

**Comment by Mr-Technician** [comment]
@cornerkey could you do some more testing? Try restarting Borderless Minecraft and also try restarting Minecraft via MultiMC if auto borderless doesn't work the first time.

**Comment by cornerkey** [comment]
@Mr-Technician i've tried restarting both multiMC , minecraft and borderless . It affects nothing.

**Comment by cornerkey** [comment]
I think multiMC starts a instance of minecraft way differently than the normal launcher does because im writing a python script to start up multiple instances of minecraft through multiMC and it is possible to launch it with windows cmd terminal by "E:\MultiMC\MultiMC.exe -launch 1.16.1INST1" < this is an example. So im guessing its something to do with it creating a virtual space of some sort for minecraft to launch. I wish i knew C+ so i could help u investigate lol. Btw i dont know if this is a valid conclusion ill do some research on my side but this is all just what i think and not fact

**Comment by Mr-Technician** [comment]
> I think multiMC starts a instance of minecraft way differently than the normal launcher does

Yes it would seem so... That's what caused the crashing issue in the first place. :(

If you want to debug further and maybe hop into a video call you can join the Discord. https://discord.gg/gbR7qba6uv

**Comment by cornerkey** [comment]
@Mr-Technician I dm'd you on discord when you are available
